### ISSUE TITLE
Non-avatar-based indiciation when a user is speaking (e.g. pulsing orb)

### ISSUE THREAD

**Issue by emclaren** [mozilla/hubs#2533]
**Is your feature request related to a problem? Please describe.**
There are so many cool humanoid avatars being created (yeah!), but many of them have very subtle mouth movements and its nearly impossible to tell which one is speaking. This makes it hard to figure out whos volume controls to press if you need to make someone louder (or mute them like mentioned in #1853)

**Describe the solution you'd like**
a setting to have a pulsing orb near the name to indicate whether a user is speaking

Also related to #1381

**Comment by emclaren** [comment]
this has been requested by the community as well a number of times. It also seems like it would impact accessibility as well

**Comment by emclaren** [comment]
Received a message from a user who referred to this ticket saying this would have helped them a lot in their experience using Hubs for the first time. They mentioned in their message they didn't want to speak because they couldn't tell if people were speaking in the room. Their message from Discord:  

"I agree with that ticket: it's hard to know when someone is talking. The suggestion in the ticket would help me then know who's volume to turn up. It would also help with knowing if someone was talking (though a mic accidentally left unmuted would just clutter that up)."

**Comment by camelgod** [comment]
I created an entity that is following the badge structure but with aframe component to show / hide. 

It works fine, but not sure about performance and proper implementation. Maybe it will inspire someone? (also sorry for only screenshots)

In hub.html (next to the other badges):
![Screenshot from 2020-07-01 13-27-11](https://user-images.githubusercontent.com/28685654/86239019-d4f0d000-bb9e-11ea-8a2e-01197b90983c.png)

In components/audio-feedback.js:
![Screenshot from 2020-07-01 13-27-36](https://user-images.githubusercontent.com/28685654/86239024-d5896680-bb9e-11ea-995c-b2074ed3fbcc.png)

Ingame looks like this:
![Screenshot from 2020-07-01 13-31-18](https://user-images.githubusercontent.com/28685654/86239220-30bb5900-bb9f-11ea-9892-8f9a33c36865.png)

**Comment by emclaren** [comment]
Received feedback from the XR Access symposium that it was often hard to see who was speaking. One person suggested adding some sort of bubble to indicate when a visitor is speaking.

**Comment by TechThomas** [comment]
@emclaren i completely agree, it seems to depend avatar to avatar whether someone can tell that someone is speaking

**Comment by emclaren** [comment]
Received a request from rem5forgood for "a more visual way to see who is talking, especially in groups of 4/5 and using different types of avatars, this can be easily lost, especially when the group doesn't know each other yet/the voices associated."

as we create more avatars with subtle mouth movements (or no mouth movements) this is increasingly necessary

**Comment by emclaren** [comment]
We received a request for this via twitter last week - https://twitter.com/merylkevans/status/1330151883995041793
